tools/travis/third-party.sh

```diff
@@ -8,8 +8,8 @@ pushd ${HOME}
 pushd 'third'
 
 # Download nltk stanford dependencies
-#stanford_corenlp_package_zip_name=$(curl -s 'https://stanfordnlp.github.io/CoreNLP/' | grep -o 'stanford-corenlp-full-.*\.zip' | head -n1)
-stanford_corenlp_package_zip_name="stanford-corenlp-full-2017-06-09.zip"
+#stanford_corenlp_package_zip_name=$(curl -s 'https://stanfordnlp.github.io/CoreNLP/history.html' | grep -o 'stanford-corenlp-full-.*\.zip' | head -n1)
```

Comment:
For context, the two issues you're trying to solve were listed in #2820. These issues were meant as "future changes" for after that PR is merged. As you might be able to see in that PR, there is a new file `tools/github_actions/third-party.sh`, which is based on this `travis` file.

We don't use Travis CI (https://travis-ci.org/) anymore, as we've replaced it with GitHub Actions. As a result, this file is no longer being used, and doesn't require updating.